cypress/integration/group1/dashboard.ts

```diff
@@ -43,4 +43,24 @@ describe('Dockstore dashboard', () => {
       .should('have.attr', 'href')
       .and('include', 'getting-started-with-services');
   });
+  it('Registering new tool through Github redirects correctly', () => {
+    cy.visit('/dashboard?newDashboard');
+    cy.contains('Tools');
+    cy.get('#registerToolButton').should('be.visible').click();
+    cy.get('#3-register-workflow-option').should('be.visible').click();
+    cy.contains('button', 'Next').should('be.visible').click();
+    cy.contains('a', 'Manage Dockstore installations on GitHub')
+      .should('have.attr', 'href')
+      .and('include', 'https://github.com/apps/dockstore-testing-application');
+  });
+  it('Registering new workflow through Github redirects correctly', () => {
```

Comment:
The blocks starting at line 47 and 57 are almost identical. You could create a function that takes a parameter(s) for what's different and avoid the duplication.